modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/timers/select-action/ActionTypeReassignment.js

```diff
@@ -36,9 +36,9 @@ const ActionTypeReassignment = (props) => {
 	const [sections, setSections] = useState([{identifier: `${Date.now()}-0`}]);
 	const ReassignmentSectionComponent = assignmentSectionComponents[section];
 
-	const [scriptSections] = useState([{identifier: `${Date.now()}-0`}]);
+	const [reassignmentSections] = useState([{identifier: `${Date.now()}-0`}]);
 
-	return scriptSections.map(({identifier}) => {
+	return reassignmentSections.map(({identifier}) => {
```

Comment:
Please remove both return and map, SelectReassignment should be returned directly. Map only occurs in sections.map.